src/averageAges.js

```diff
@@ -15,11 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
+    ? (genderIs(person, 'm') && fromCentury(person, century))
+    : (genderIs(person, 'm')));
+
+  return avarageAge(men);
+}
+
+function genderIs(person, sex) {
+  return person.sex === sex;
+}
```

Comment:
I would say this one is redundant, it looks pretty good without this custom helper